addon/templates/components/sl-grid.hbs

```diff
@@ -0,0 +1,126 @@
+<header class="grid-header">
+    <div class="row">
+        <div class="col-sm-6">
+            {{yield}}
+        </div>
+
+        {{#if filterPath}}
+            <div class="col-sm-6 text-right">
+                {{#sl-button
+                    action="toggleFilterPane"
+                    size="small"
+                }}
+                    {{filterButtonLabel}}
+                {{/sl-button}}
+            </div>
+        {{/if}}
+    </div>
+</header>
+
+<div class="filter-pane">
+    {{#if filterPaneOpen}}
+        <div class="filter-content">
+            {{#if filterPath}}
+                {{render-dynamic filterPath}}
+            {{/if}}
+        </div>
+    {{/if}}
+</div>
+
+<div class="list-pane">
+    <table class="table column-headers">
+        <thead>
+            <tr>
+                {{#each column in columns}}
+                    {{sl-grid-column-header column=column onClick="sortColumn"}}
+                {{/each}}
+
+                {{#if showActions}}
+                    <th class="actions-cell"></th>
+                {{/if}}
+            </tr>
+        </thead>
+    </table>
+
+    <div class="content">
+        <table class="table table-hover table-striped">
+            <tbody>
+                {{#each row in content itemController=detailPath}}
```

Comment:
we should phase out use of itemController as supposedly...."In all of these cases, we did significant work in the 1.x series, especially in 1.12 and 1.13, to make sure that the dominant use-cases for these features were addressed by existing or new features." (http://emberjs.com/blog/2015/05/24/another-two-oh-status-update.html#toc_removals)